### DISCUSSION TITLE
How do I make scrolling smoothly between blocks?

### DISCUSSION THREAD

**Discussion by BloodTusk02** [idiotWu/smooth-scrollbar#442] [Q&A]
I have 4 blocks in the html document, and I want to scroll smoothly without being able to stop in the middle, i.e. between blocks.
The website I gave as an example implements what I am trying to do.
https://www.prevint.pt/en/
Please help!

**Comment by dolphin-wood** [comment]
Instead of this scrollbar plugin, I think maybe you need a slider plugin like [splide](https://github.com/Splidejs/splide)?

**Reply by BloodTusk02** [reply]
Will I not be able to achieve what I want with this particular plugin?

**Reply by dolphin-wood** [reply]
It would be quite difficult – you have to implement the slider logic yourself, for example:

```
# pseudo code

@debounce(100)
fn wheelScroll(delta):
  if delta > 0:
    goto next block

  if delta < 0:
    goto previous block
```

**Reply by BloodTusk02** [reply]
I see, then it would be easier to use another plugin. Thanks for the answers! Good luck)
